orchagent/saihelper.h

```diff
@@ -4,10 +4,13 @@
 
 #include <string>
 #include "orch.h"
+#include "producertable.h"
+#include <sairedis.h>
```

Comment:
this should be externed C

Reply:
I will move it to the `extern C` block in `saihelp.c`. For other cpp files, they do not need this header.

Reply:
fixed